docs/marketplace/dev/feature-integration-guides/202200.0/merchant-portal-marketplace-customer-specific-prices-feature-integration.md

```diff
@@ -0,0 +1,196 @@
+| MerchantRelationshipPriceProductMapperPlugin | Maps merchant relationship data to `PriceProductTableViewTransfer` transfer object. |  | Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui |
+
+<details>
+<summary markdown='span'>src/Pyz/Zed/ProductMerchantPortalGui/ProductMerchantPortalGuiDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\ProductMerchantPortalGui;
+
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductAbstractTableConfigurationExpanderPlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductConcreteTableConfigurationExpanderPlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductMapperPlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductTableFilterPlugin;
+use Spryker\Zed\ProductMerchantPortalGui\ProductMerchantPortalGuiDependencyProvider as SprykerProductMerchantPortalGuiDependencyProvider;
+
+class ProductMerchantPortalGuiDependencyProvider extends SprykerProductMerchantPortalGuiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductTableFilterPluginInterface>
+     */
+    protected function getPriceProductTableFilterPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductTableFilterPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductAbstractTableConfigurationExpanderPluginInterface>
+     */
+    protected function getPriceProductAbstractTableConfigurationExpanderPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductAbstractTableConfigurationExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductConcreteTableConfigurationExpanderPluginInterface>
+     */
+    protected function getPriceProductConcreteTableConfigurationExpanderPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductConcreteTableConfigurationExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductMapperPluginInterface>
+     */
+    protected function getPriceProductMapperPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductMapperPlugin(),
+        ];
+    }
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
+
+Log in to the Merchant Portal with a merchant that has at least one merchant relationship.
```

Comment:
Please check all the verification steps and plugin specifications with Gramarly.